VTEX - Orders API.json

```diff
@@ -19944,6 +19947,16 @@
                                 "type": "number",
                                 "description": "Unit multiplier for item update. For example, if you wish to increase an order with three times more items than a single one, you would fill in with `3.0`.",
                                 "example": 3.0
+                              },
+                              "manualDiscountValue": {
+                                "type": "integer",
+                                "description": "This field shows if a manual price was applied to the total order amount. The value is in cents and starts from 0.",
+                                "example": 0
+                              },
+                              "manualIncrementValue": {
+                                "type": "integer",
+                                "description": "This field shows if an increment value was applied to the total order amount. The value is in cents and starts from 0.",
```

Comment:
I don't understand what "starts from 0" mean. Does it mean this value is necessarily equal or greater than 0? I recommend being clearer here.